onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -168,6 +168,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "Hidden dimension of Q, K, V: hidden_size, hidden_size and v_hidden_size",
               AttributeProto::INTS,
               OPTIONAL_VALUE)
+        .Attr("kv_cache_past_present",
```

Comment:
Maybe name it as reuse_cache_buffer, which means the buffer for the past input is reused for the present output.

BTW, we may not need this attribute. Instead, we rely on whether past_sequence_length input exists for such flag.

Reply:
Nice suggestion.

Reply:
Will still use this variable, just change name, the past_sequence_length may better only control its own data for future expansion.